### PR TITLE
Renamed my css file to james.css

### PR DESCRIPTION
**Pull request by jsa-codes**
Renamed my CSS file to "james.css"

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,10 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <!-- Russ' Stylesheet -->
-    <link rel="stylesheet" href="/path/to/obnoxious.css" />
-    <link rel="stylesheet" href="main.css" />
+    <!-- <link rel="stylesheet" href="/path/to/obnoxious.css" /> -->
```
Comment (Russ716) [inline index.html:8]: Are these comments needed or can be deleted?

File: james.css [hunk 2]
```diff
@@ -89,7 +78,7 @@ footer h1 {
     font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
 }
 
-html,
+/* html,
```
Comment (Russ716) [inline james.css]: why comment instead of remove?

File: index.html (hunk 1 above)
Comment (jsa-codes) [reply index.html:8]: Let's leave it for now...just for testing purposes.

File: james.css (hunk 2 above)
Comment (jsa-codes) [reply james.css]: Can be removed.

**Review by Russ716 — APPROVED** [review]

**Review by carlydopps — APPROVED** [review]

**Review by justindority — APPROVED** [review]
Looks good!

**Review by mandapandaa8 — APPROVED** [review]

**Pull request merged**